qiskit_addon_dice_solver/dice_solver.py

```diff
@@ -595,13 +595,12 @@ def _construct_ci_vec_from_addresses_amplitudes(
 
     addr_map = {uni_addr: i for i, uni_addr in enumerate(uniques)}
 
-    for idx, address in enumerate(addresses):
```

Comment:
I'm surprised the linter didn't complain about the unused `idx` variable here

Reply:
Oops, I see it below now :)

Reply:
It was used, in line 604.